plenum/server/node.py

```diff
@@ -260,6 +260,7 @@ def __init__(self,
                                notifierEventsEnabled=self.config.SpikeEventsEnabled)
 
         self.replicas = self.create_replicas()
+        self.replicas.set_default_handlers()
```

Comment:
As long as notify functionality is mandatory I think it would be better to move this call to Replicas constructor or at least to self.create_replicas() function.

Reply:
Moved